docs/source/model_design/vivarium_overview/vivarium_versus_other_model_types/index.rst

```diff
@@ -94,61 +94,90 @@ expands on what differentiates IBM, including microsimulation, from other types
+risk factors that affect different CVD outcomes, including: body mass index (BMI), systolic blood pressure (SBP), low-density lipoprotein cholesterol levels (LDL-C), and fasting 
+plasma glucose (FPG). Using NHANES survey data, we calculated correlation coefficients for these relationships, which could then be used to generate propensity values for all
+simulants. As such, the correlation coefificients remain constant as simulants age and their risk exposure values change. You can read more about our modeling strategy for 
+these risk factor correlations :ref:`on this page <2023_sbp_ldlc_fpg_bmi>`.
 
 Continuous values
 ~~~~~~~~~~~~~~~~~
 
-e.g., allows the modeling of joint distribution between blood pressure and blood glucose levels -- this kind of thing is hard with categorical data characteristic of multiplication models
+Similarly, microsimulation allows the modeling of joint distribution between continuous values, something which is not as possible with categorical data characteristic of 
+multiplication models.
+
+For example, in our intravenous (IV) iron intervention microsimulation, we model the joint distribution of maternal BMI and hemoglobin levels, as two risk exposures associated with poor
+health outcomes for both mother/birthing parent and child. We treat hemoglobin as a continuous risk exposure that varies as simulants age and move through the pregancy model states,
+while maternal BMI is a categorical risk exposure conditional on hemoglobin status (i.e., above or below 100 g/L) at the start of the pregnancy. Read more about the modeling strategy
+we used for this joint distribution :ref:`here <2019_risk_exposure_maternal_bmi_hgb>`.
+
+.. todo::
+
+    Check what we agreed is a good BLISS alternative for 'maternal BMI'. 
 
 Time-varying rates dependent on multiple factors
 ~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
-e.g., Multiple Myeloma model
+Another complexity enabled by microsimulation is the inclusion of time-varying rates dependent on multiple factors. A good example of this is in our microsimulation of Multiple Myeloma, 
+which allow relapse hazard rates to vary over time since treatment line initiation. Read more about how we calculated relapse hazard rates :ref:`here <calc_mortality_relapse_hazard>`.
+
+.. todo:: 
+
+    Elaborate on example from Multiple Myeloma model
 
 Incorporating uncertainty
 ~~~~~~~~~~~~~~~~~~~~~~~~~
 
-Our microsimulation framework is formulated to routinely propagate forward the uncertainty captured by these draws. Monte Carlo methods allow us to effectively incorporate different kinds of uncertainty into our models. 
+Microsimulation methods provide a useful approach to modeling stochastic uncertainty in a conceptually valid manner because chance – that is, reliance on randomly generated numbers – 
+is used to govern movement between health states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple 
```

Comment:
```suggestion
is used to govern movement between states for individual simulants from a single computer-generated population, as opposed to comparing the role of chance across multiple 
```
Since microsimulation is not specific to health.